commands/eval/classes/run_object.py

```diff
@@ -0,0 +1,78 @@
+import re
```

Comment:
I extracted classes into separate file as I ended up with cyclical dependencies during testing. The functions are old here, the class is new